packages/ui-library/src/components/sidebar-navigation/mobile.js

```diff
@@ -10,7 +10,7 @@ import { useNavigationContext } from "./index";
  * @param {string} [closeButtonScreenReaderText] The close button screen reader text.
  * @returns {JSX.Element} The mobile element.
  */
-const Mobile = ( { children, openButtonScreenReaderText = "Open", closeButtonScreenReaderText = "Close" } ) => {
+const Mobile = ( { children, openButtonScreenReaderText = "Opens a menu", closeButtonScreenReaderText = "Closes a menu" } ) => {
```

Comment:
This affects only the defaults. Which maybe we should not provide and just make these props required. As maybe they are too easy to forget to provide in translated form if we do this. What do you think?

Either way, in this case you should change the [Settings' implementation props](https://github.com/Yoast/wordpress-seo/blob/trunk/packages/js/src/settings/app.js#L232:L233) to actually have an effect there.

Reply:
Removed default values from ButtonScreenReader props and set those props as required.
Checked screen reader is catching Buttons as `Open sidebar` and `Close sidebar`.